app/views/guests/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= @guest.name %></h1>
+
+<ul id='rooms'>
+	<% @rooms.each do |room| %>
+		<li>
+			<%= room.suite %>: <%= room.rate %> - <%= room.hotel.name %>
+		</li>
+	<% end %>
+</ul>
+
+<%= render partial: 'select_room', locals: {
+	uri: "/room_guests/#{@guest.id}",
```

Comment:
Nice job making this partial! For these ICs it's usually not necessary but this is good practice.